src/Pim/Bundle/DataGridBundle/Datagrid/Configuration/Product/SortersConfigurator.php

```diff
@@ -53,6 +54,10 @@ protected function addAttributeSorters()
             sprintf('[%s]', FormatterConfiguration::COLUMNS_KEY)
         );
         foreach ($attributes as $attributeCode => $attribute) {
+            if (AttributeTypes::IDENTIFIER === $attribute['type']) {
```

Comment:
shouldn't we have a dedicated sorter like we did for the filter?
https://github.com/jjanvier/pim-community-dev/blob/TIP-613-unified/src/Pim/Bundle/CatalogBundle/Elasticsearch/Filter/IdentifierFilter.php